session/bootstrap.go

```diff
@@ -365,6 +365,17 @@ const (
 		oldReadLease bigint(20) NOT NULL DEFAULT 0,
 		PRIMARY KEY (tid)
 	);`
+	// CreateAnalyzeOptionsTable stores the analyze options used by analyze and auto analyze.
+	CreateAnalyzeOptionsTable = `CREATE TABLE IF NOT EXISTS mysql.analyze_options (
+		table_id BIGINT(64) NOT NULL,
+        sample_num BIGINT(64),
+        sample_rate DOUBLE NOT NULL DEFAULT -1,
+        buckets BIGINT(64),
+        topn BIGINT(64),
+        column_choice enum('UNDEFINED','ALL','PREDICATE','LIST') NOT NULL DEFAULT 'UNDEFINED',
+        column_ids varchar(1024),
```

Comment:
The max limit of the column number is 4096. Maybe the max length of varchar needs to be at least `len("1,2,3...,4095,4096")=19372`?